### ISSUE TITLE
Re-enable fejta-bot

### ISSUE THREAD

**Issue by cdrage** [helm/helm#7175]
Output of `helm version`: N/A

Output of `kubectl version`: N/A

Cloud Provider/Platform (AKS, GKE, Minikube etc.): N/A

@fejta-bot has been useful in the past with regards to issues getting out of hand. There's over 880 open issues now, perhaps this should be re-enabled?

**Comment by bacongobbler** [comment]
I've advocated against the stale issue bot in the past. Often, feature requests are left open for a community member to implement. Over time, we were marking more issues as "keep open" than letting them go stale. The stale issue bot became more of a hindrance than a helpful tool.

On that same note, often times an issue can go stale without a true resolution. This leads to people requesting for an issue to be re-opened, or they open a new ticket. This makes it more difficult to follow the conversation when it's spread across multiple tickets.

In other words, there hasn't been a compelling reason why we should re-enable the stale issue bot based on past experiences. However, it's always good to reflect and determine if that argument still stands.

Perhaps the other core maintainers have an opinion on this?

For the time being, if there are specific issues we should close, please feel free to comment on them and mention why they should be closed.

**Comment by cdrage** [comment]
IMO my reasoning is that there are over 400+ support issues opened. Most of which could be closed using the bot.

A lot of which are seldom closed after a user has found a solution. The inevitable: 

![image](https://user-images.githubusercontent.com/6422176/70356071-916cf800-1841-11ea-89ed-412d1c1a1d0c.png)

**Comment by mattfarina** [comment]
In the Helm project we have done this on a repo by repo basis. For example, the charts repo uses Probots stale issue closer. On Helm I suspect we would use Probot instead of Feta bot.

That being said, we are very  slow to close issues that are support ones. We should do something about those but we have been tentative in adding automation that closes out issues prematurely. If we want to do something we should talk about it at a developer call first.

**Comment by bacongobbler** [comment]
I spent some time in the issue queue today, triaging [anything labeled as question/support](https://github.com/helm/helm/issues?q=is%3Aissue+is%3Aopen+label%3Aquestion%2Fsupport).

The end result: ~260 issues were closed. Many questions were already answered and the author forgot to close the issue, or the core maintainers had responded and were waiting for a response from the author.

After going through the queue, I'm still apprehensive about re-introducing a stale issue bot to the project. There were many cases where tickets from 2.5 years ago were still relevant today. Re-enabling the stale issue bot means the core maintainers now have to go back and mark these tickets as "freeze" to ensure these tickets are not automatically closed out. I fear issues labeled as a proposal or a feature request would require more nuance to identify which should be frozen and which should be closed out. The end result still being that the core maintainers end up with more maintenance work, and users feel frustrated that their proposals are closed out after 3 months.

In the meantime, we're back down to ~140 open issues labeled question/support, which is a lot more reasonable compared to the rest of the queue.
